unit-tests/unit-test-default-main.cpp

```diff
@@ -0,0 +1,29 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#define CATCH_CONFIG_RUNNER
+#include <unit-tests/catch/catch.hpp>
+#include <string>
+
+extern std::string context;
+
+using namespace Catch::clara;
+
+int main(int argc, char* argv[]) {
```

Comment:
Please provide an overview of all the custom options we accept, or where to find a description.

Reply:
Also, if there's any easy way to further customize it, give an example -- or, if someone would want to replace it, also provide an example of how.

Reply:
Not sure what you mean. At the moment the only option handed over to the tests is `context`

Reply:
Right... so please write documentation explaining each of our custom options.
My second comment is asking for documentation of extensions and/or replacements to this mechanism we've built here.